plane/source/docs/guide-rudder-only-plane.rst

```diff
@@ -0,0 +1,125 @@
+The rudder servo should be attached to the :ref:`RCMAP_YAW<RCMAP_YAW>` channel as well (normally channel 4).
+
+In the Radio Calibration setup page, you should now observe that a right roll command on the sticks results in the same
+movement as a right yaw command.
+
+Later, when you set up :ref:`arming-your-plane`, remember that you can now arm with the configured roll stick.
+
+After the RC inputs are configured, configure the outputs.
+
+.. warning:: Make sure the :ref:`AHRS_ORIENTATION<AHRS_ORIENTATION>` is set correctly for the autopilot. If it is incorrect, 
+             this setup will fail, and the plane may crash upon entry into any stabilize mode.
+             
+Servo cables can be connected to any output of the autopilot,
+but using the default channels 2-4 listed below is recommended. 
+Set the SERVOn_FUNCTION to the appropriate values
+
+You can do this by setting the  parameter directly using Ground Station software such as Mission Planner or QGroundControl using either the full parameter lists or setup tab for outputs.
+In Mission Planner you can do this on the SETUP -> Mandatory Hardware -> Servo Output page. Each channel has a dropdown which can be used to select the function of the channel.
+
+.. raw:: html
+
+   <table border="1" class="docutils">
+   <tr><th>Parameter</th><th>Value</th><th>Meaning</th></tr>
+   <tr><td>SERVO2_FUNCTION</td><td>19</td><td>elevator</td></tr>
+   <tr><td>SERVO3_FUNCTION</td><td>70</td><td>throttle</td></tr>
+   <tr><td>SERVO4_FUNCTION</td><td>21</td><td>rudder</td></tr>
+   </table>
+
+
+Servo Function & Reversal
+=========================
+
+The next step is to correct the servo reversals. 
+Connect the battery (with propeller removed) and turn on the RC transmitter.
+Switch to FBWA mode using the function switch or a ground station command, and 
+disable the safety switch (if fitted). We use FBWA for setup because it prevents
+double-reversing inputs and outputs. Double-reversing causes correct manual
+behavior, but dangerous and destabilizing behavior in other flight modes.
+
+When the plane is level, the servos should be near their trim (neutral) values. 
+Move the plane and leave the transmitter sticks centered while 
+monitoring the control surfaces to determine if the reversals are correct.
+See the table for the correct control surface response to the movements. 
+In each instance, the plane should move its control surfaces to level itself
+and coordinate its turns.
+
+.. raw:: html
+
+   <table border="1" class="docutils">
+   <tr><th>Movement</th><th>Action</th></tr>
+   <tr><td>Roll Plane Right</td><td>Rudder moves left</td><tr>
+   <tr><td>Roll Plane Left</td><td>Left aileron moves down and right aileron moves up</td><tr>
+   <tr><td>Pitch plane up</td><td>Elevator moves down</td></tr>
+   <tr><td>Pitch plane down</td><td>Elevator moves up</td></tr>
+   <tr><td>Yaw Plane Right</td><td>Rudder moves left</td></tr>
```

Comment:
again there  is no default yaw stabilization....only roll should move the rudder